ipwhois/rdap.py

```diff
@@ -551,7 +551,8 @@ def parse(self):
 
             try:
 
-                self.vars[v] = self.json[v].strip()
+                if self.json[v]:
+                    self.vars[v] = self.json[v].strip()
 
             except (KeyError, ValueError):
```

Comment:
I'm facing the same issue, and I think fixing this by handling the AttributeError in the except block is probably better, given the structure of the current implementation.